src/AbstractServer.php

```diff
@@ -67,10 +67,10 @@ protected function _buildCallback(Reflection\AbstractFunction $reflection): Meth
      * @return Method\Definition
      * @throws Exception\RuntimeException on duplicate entry
      */
-    // @codingStandardsIgnoreStart
+    // phpcs:disable
```

Comment:
I'd do two things with this method:

- Remove the underscore prefix (per the docblock!)
- Make the method `final` (so it cannot be overridden by end-users)

This will allow getting rid of the phpcs dance, and also fulfill the ideas already sketched for 3.0.